frontend/app/src/composables/api/task.ts

```diff
@@ -58,6 +58,9 @@ export function useTaskApi() {
       else if (statusCode === 400) {
         throw new ApiValidationError(message);
       }
+      else if (statusCode === 502) {
+        throw new Error(message);
+      }
```

Comment:
where did we get 502?

Reply:
os, a backend change was merged, and the asset repo which included the file was not yet merged, so, when I rebased, I was getting this 502 error, it is also documented on the endpoint.
so, I added a case to handle it.